node/src/network/indexer_heights.rs

```diff
@@ -23,6 +24,13 @@ impl IndexerHeightTracker {
         if height > current {
             atomic.store(height, std::sync::atomic::Ordering::Relaxed);
         }
+
+        if let Err(e) = metrics::PEERS_INDEXER_HEIGHTS
+            .get_metric_with_label_values(&[&participant.raw().to_string()])
+            .map(|x| x.set(height as i64))
```

Comment:
Can we use something more descriptive instead of `x`.

```suggestion
            .map(|gauge| gauge.set(height as i64))
```